src/tests/snapshots/cargo_vet__tests__audit_as_crates_io__simple-audit-as-crates-io.snap

```diff
@@ -2,8 +2,11 @@
 source: src/tests/audit_as_crates_io.rs
 expression: output
 ---
-error: some non-crates.io-fetched packages match published crates.io versions
-  first-party:10.0.0
 
-All of these must have a `policy.*.audit-as-crates-io` entry
+  × There are some issues with your policy.audit-as-crates-io entries
+
+Error: 
+  × Some first-party packages match published crates.io versions
+  │   first-party:10.0.0
+  help: Add a `policy.*.audit-as-crates-io` entry for them
```

Comment:
The change in this PR reverted https://github.com/mozilla/cargo-vet/commit/a00b95ae863eb214f7d907c05ae0475a4079e876, possibly unintentionally?

Reply:
Good catch, I just pushed https://github.com/mozilla/cargo-vet/commit/aaaf4f7b964c89ac7e6e7551b37a6bbe71be1e00 to switch it back.